frontend/src/components/note/NoteCard.vue

```diff
@@ -1,22 +1,53 @@
 <script setup lang="ts">
 import { onMounted, ref } from 'vue';
 
-import { Card, Divider } from '@/lib/primevue';
-import { userService } from '@/services';
+import { Button, Card, Divider, useToast } from '@/lib/primevue';
+import { noteService, userService } from '@/services';
 import { formatDate, formatDateShort } from '@/utils/formatters';
 
 import type { Ref } from 'vue';
 import type { Note } from '@/types';
+import NoteModal from './NoteModal.vue';
 
 // Props
 type Props = {
   note: Note;
 };
 
 const props = withDefaults(defineProps<Props>(), {});
+const toast = useToast();
```

Comment:
I've generally considered `useToast` and `useConfirm` to be actions and placed them as the first things under the `// Actions` comment. Could we update this for consistency?